src/docs/product/sentry-basics/search/index.mdx

```diff
@@ -12,6 +12,12 @@ Search is available on several features throughout [sentry.io](https://sentry.io
 
 ## Query Syntax {#syntax}
 
+<Alert level="info" title="Important">
+
+For organizations participating in the limited release, the transaction filter on the performance page has been simplified to support metric enhanced performance. The search bar can only be used to filter by transaction name. See [transaction search](/product/performance/filters-display/#transaction-search).
+
+</Alert>
+
```

Comment:
I think it would be good to deemphasize this by moving it lower in the document. The transaction search limitation is only in one place, and it only affects the raw string! No need to scare people at the very top of the page. Maybe this could live under the "The token `example error`" paragraph?

Reply:
I was mixed between putting it at the top, the reason I put it at the top is because it is what you see when you click "read the docs" in the search bar.
![image](https://user-images.githubusercontent.com/44422760/183485858-c8fdce20-f1d7-442c-af3e-b3daf70814e0.png)

So it might be confusing for a user, if they click read the docs while searching for a transaction, only to figure out later in the documentation that the query syntax doesn't apply.